specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/machineLearningServices.json

```diff
@@ -1500,6 +1582,11 @@
             "$ref": "#/definitions/SharedPrivateLinkResource"
           },
           "description": "The list of shared private link resources in this workspace."
+        },
+        "notebookInfo": {
```

Comment:
This notebookInfo is a new property that will be ignored by current client. So it's not a breaking change to our service